File: iOS/AppFactory/AppFactory/Http/Enum/GallerySortType.swift
//
//  GallerySortType.swift
//  AppFactory
//
//  Created by Leapfrog-Software on 2018/04/09.
//  Copyright © 2018年 Leapfrog-Inc. All rights reserved.
//

enum GallerySortType {
    case evaluate
    case cost
    
    func toString() -> String {
        switch self {
        case .evaluate:
            return "0"
        case .cost:
            return "1"
        }
    }
}

File: Android/app/src/main/java/leapfrog_inc/appfactory/Http/Enum/GallerySortType.java
package leapfrog_inc.appfactory.Http.Enum;

/**
 * Created by Leapfrog-Software on 2018/04/16.
 */

public enum GallerySortType {
    evaluate,
    cost;

    public String toValue() {

        switch (this) {
            case evaluate:
                return "0";
            case cost:
                return "2";
            default:
                return "";
        }
    }
}
